packages/api/api.go

```diff
@@ -180,15 +175,15 @@ func DefaultHandler(method, pattern string, params map[string]int, handlers ...a
 				errorAPI(w, `E_RECOVERED`, http.StatusInternalServerError)
 			}
 		}()
+
 		w.Header().Set("Access-Control-Allow-Origin", "*")
 		w.Header().Set("Content-Type", "application/json; charset=utf-8")
-		if !IsInstalled() && r.URL.Path != `/api/v2/install` {
-			if model.DBConn == nil && !config.IsExist() {
-				errorAPI(w, `E_NOTINSTALLED`, http.StatusInternalServerError)
-				return
-			}
-			Installed()
+
+		if !conf.WebInstall && r.URL.Path != `/api/v2/install` {
```

Comment:
i do not understand why we are checking some WebInstall flag, i think we should check config existence.
If config.toml exists than it already installed, if not it is not installed.
What if we do installation through console?